configdb/hardware/management/commands/init_e2e_data.py

```diff
@@ -24,8 +24,10 @@ def add_arguments(self, parser):
                             help='Site latitude in degrees')
         parser.add_argument('--longitude', required=True, type=float,
                             help='Site longitude in degrees')
+        parser.add_argument('--instrument-state', dest='instrument_state', type=str, default='MANUAL',
```

Comment:
Not a big deal, but maybe pull the state from what's set on the `Instrument` model for the default here instead?

Reply:
I actually made the default here different than the instrument models default because I wanted to keep it the same as what it was before, so the e2e tests didn't need to be modified (They assume it is manual to schedule their observations I think)